frontend/src/pages/friendDashboard.tsx

```
@@ -1,99 +1,135 @@
-          <p className="text-xl font-semibold">Stay Safe, Download Our App</p>
-          <div className="flex justify-center mt-4 gap-4">
-            <button className="px-6 py-3 bg-green-800 text-white rounded-lg hover:bg-green-600 transition duration-300 shadow">
-              App Store
-            </button>
-            <button className="px-6 py-3 bg-green-800 text-white rounded-lg hover:bg-green-600 transition duration-300 shadow">
-              Google Play
-            </button>
+        {/* Safety Status Card */}
+        <div className="mt-6 bg-white rounded-xl shadow-sm border border-gray-200 p-6">
+          <h3 className="text-xl font-semibold text-gray-900 mb-4">Safety Status</h3>
+          <div className="flex items-center gap-2" role="status" aria-label="Safety status">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)